### PR TITLE
First API testing

### PR DESCRIPTION
**Pull request by Maosorio92**

### CODE REVIEW

File: test/MyFirstApiConsume.test.js
Comment (danielgalvis98) [inline test/MyFirstApiConsume.test.js]: The idea of ITs is to have them inside the describe, it is a section. (Same for the other its)

File: test/MyFirstApiConsume.test.js
Comment (danielgalvis98) [inline test/MyFirstApiConsume.test.js]: Avoid this empty lines at the end of functions or methods

File: test/MyFirstApiConsume.test.js
Comment (danielgalvis98) [inline test/MyFirstApiConsume.test.js]: Avoid empty line

File: test/MyFirstApiConsume.test.js
Comment (danielgalvis98) [inline test/MyFirstApiConsume.test.js]: Empty lines

File: test/MyFirstApiConsume.test.js
Comment (danielgalvis98) [inline test/MyFirstApiConsume.test.js]: lines

File: test/MyFirstApiConsume.test.js
Comment (danielgalvis98) [inline test/MyFirstApiConsume.test.js]: lines

File: test/MyFirstApiConsume.test.js
Comment (danielgalvis98) [inline test/MyFirstApiConsume.test.js]: lines

File: test/MyFirstApiConsume.test.js
Comment (danielgalvis98) [inline test/MyFirstApiConsume.test.js]: lines

File: test/MyFirstApiConsume.test.js
Comment (danielgalvis98) [inline test/MyFirstApiConsume.test.js]: This last line of the file should not have any spaces or any character. Last line should be line 89, but it should be empty, only being created by the line break, which represents the end of file

Comment (Maosorio92) [reply test/MyFirstApiConsume.test.js]: Done!

File: test/MyFirstApiConsume.test.js
Comment (Maosorio92) [reply test/MyFirstApiConsume.test.js]: Done!

File: test/MyFirstApiConsume.test.js
Comment (Maosorio92) [reply test/MyFirstApiConsume.test.js]: Done!

File: test/MyFirstApiConsume.test.js
Comment (Maosorio92) [reply test/MyFirstApiConsume.test.js]: Done!

File: test/MyFirstApiConsume.test.js
Comment (Maosorio92) [reply test/MyFirstApiConsume.test.js]: Done!

File: test/MyFirstApiConsume.test.js
Comment (Maosorio92) [reply test/MyFirstApiConsume.test.js]: Done!

File: test/MyFirstApiConsume.test.js
Comment (Maosorio92) [reply test/MyFirstApiConsume.test.js]: Done!

File: test/MyFirstApiConsume.test.js
Comment (Maosorio92) [reply test/MyFirstApiConsume.test.js]: Done!

File: test/MyFirstApiConsume.test.js
Comment (Maosorio92) [reply test/MyFirstApiConsume.test.js]: Done!

**Comment by manuelq12** [comment]
@Maosorio92 Hey man, just a quick question, why do you need the console logs?
The idea is that assertions show you the results you receive, so

**Comment by Maosorio92** [comment]
@manuelq12 Hello!
I can erase them, it is just to see that everything is going well, 
I will erase them now
Thank you!

**Comment by Maosorio92** [comment]
@manuelq12 
I have already erased console logs in Rev3.
Thank you! I wait for your approval to continue with next step.

**Review by manuelq12 — APPROVED** [review]
LGTM 🎖️

**Review by danielgalvis98 — APPROVED** [review]

**Pull request merged**
